src/Pwa/Controller/PageController.php

```diff
@@ -102,6 +111,10 @@ private function getPageResult(PageLoaderInterface $pageLoader, PageLoaderContex
         $pageResult->setResourceIdentifier($pageLoaderContext->getResourceIdentifier());
         $pageResult->setCanonicalPathInfo($pageLoaderContext->getRoute()->getCanonicalPathInfo() ?: $pageLoaderContext->getRoute()->getPathInfo());
 
+        $canonicalPathInfo = $pageLoaderContext->getRoute()->getCanonicalPathInfo();
+        $ressourceIdentifier = $pageLoaderContext->getResourceIdentifier();
+        $this->eventDispatcher->dispatch(new PageLoaderLoadedEvent($pageResult, $pageLoaderContext, $request, $canonicalPathInfo, $ressourceIdentifier));
```

Comment:
You don't explicitly have to pass the request all the way, as it is part of the `$pageLoaderContext`.